persistent-template.yml

```diff
@@ -912,6 +912,10 @@ objects:
           image: projectodd/action-python-2:${PROJECTODD_VERSION}
           imagePullPolicy: IfNotPresent
           command: ["/bin/sh", "-c", "echo", "success"]
+        - name: preload-openwhisk-php7
```

Comment:
iirc, we hit some max on the number of init containers we could have here, but if you saw it work in a virgin minishift, i guess not anymore.

Reply:
I haven't seen any immediate failures while recreating the project with this in the template. I assume that it should fail at this stage, so I think we're fine.